omnigibson/sensors/sensor_base.py

```diff
@@ -73,24 +73,25 @@ def get_obs(self):
         if not self._enabled:
             return dict()
 
-        obs = self._get_obs()
```

Comment:
Can you make sure that all docstrings / methods of `get_obs()` / `_get_obs()` all have the same return type (2-tuple of obs dict and info dict)? I believe `GymObservable` class needs to get updated as well.